nntrainer/tensor/basic_planner.cpp

```diff
@@ -30,8 +30,10 @@ size_t BasicPlanner::planLayout(
   const std::vector<size_t> &memory_size,
   const std::vector<std::pair<unsigned int, unsigned int>> &memory_validity,
   std::vector<size_t> &memory_offset) const {
+
```

Comment:
Just checking, the commit message and the content seems not matching, so if it's okay?

```
[memoryPool] Implementation for memory pool
This patch provides implementation for memory pool which uses a memory
planner to plan the layout of the memory and provides efficiency of the
given planner.

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>
```

Reply:
this is because of the rebase I guess. I will remove them manually.

Reply:
Cool :)